{"name": "test_login_with_valid_email_and_invalid_password", "status": "passed", "start": 1684490427461, "stop": 1684490429292, "uuid": "357d739f-5367-41a3-84a5-7650b670f0f3", "historyId": "0f7f188bae322a777580df5f4c12caa1", "testCaseId": "0f7f188bae322a777580df5f4c12caa1", "fullName": "tests.test_Login.TestLogin#test_login_with_valid_email_and_invalid_password", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-VO28CS3"}, {"name": "thread", "value": "22428-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_Login"}]}